### PR TITLE
[DOC-2829] Update triggers doc with note about Harness variable expression support

### PR DESCRIPTION
**Pull request by brian-f-harness**
Update triggers doc with a note about Harness variable expression support

For reviewers, previews are available:

https://64a6075675213216f2bc5089--harness-developer.netlify.app/docs/platform/triggers/schedule-pipelines-using-cron-triggers/

https://64a6075675213216f2bc5089--harness-developer.netlify.app/docs/platform/triggers/trigger-on-a-new-artifact/

https://64a6075675213216f2bc5089--harness-developer.netlify.app/docs/platform/triggers/trigger-pipelines-on-new-helm-chart/

https://64a6075675213216f2bc5089--harness-developer.netlify.app/docs/platform/triggers/triggering-pipelines/

https://64a6075675213216f2bc5089--harness-developer.netlify.app/docs/platform/triggers/trigger-deployments-using-custom-triggers/

https://64a6075675213216f2bc5089--harness-developer.netlify.app/docs/platform/triggers/selective-stage-execution-using-triggers/

https://64a6075675213216f2bc5089--harness-developer.netlify.app/docs/platform/triggers/custom-trigger-passing-data/

https://64a6075675213216f2bc5089--harness-developer.netlify.app/docs/platform/triggers/trigger-pipelines-from-a-third-party-ci-system/

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [x] Maintenance/Chore

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

**Review by shivamnegi94 — DISMISSED** [review]

File: docs/platform/11_Triggers/trigger-on-a-new-artifact.md [hunk 1]
```diff
@@ -95,7 +99,7 @@ You can also set tag as a runtime input and then use `<+trigger.artifact.build>`
 
    ![](./static/trigger-on-a-new-artifact-24.png)
 
-   You reference an artifact in the Stage's Service Definition in your manifests using the expression `<+artifact.image>`. See [Add Container Images as Artifacts for Kubernetes Deployments](/docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-kubernetes-category/add-artifacts-for-kubernetes-deployments/).
+   You reference an artifact in the Stage's Service Definition in your manifests using the expression `<+artifact.image>`. Go to [Add Container Images as Artifacts for Kubernetes Deployments](/docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-kubernetes-category/add-artifacts-for-kubernetes-deployments/).
```
Comment (roshnisarangadharan) [inline docs/platform/11_Triggers/trigger-on-a-new-artifact.md:102]:
```suggestion
   You reference an artifact in the stage's service definition in your manifests using the expression `<+artifact.image>`. Go to [Add Container Images as Artifacts for Kubernetes Deployments](/docs/continuous-delivery/deploy-srv-diff-platforms/kubernetes/cd-kubernetes-category/add-artifacts-for-kubernetes-deployments/) for more details.
```

File: docs/platform/11_Triggers/trigger-pipelines-on-new-helm-chart.md [hunk 2]
```diff
@@ -76,7 +80,7 @@ Typically, you add a Helm Chart Trigger to a Pipeline that deploys a Helm Chart.
 
    ![](./static/trigger-pipelines-on-new-helm-chart-06.png)
 
-   See [Helm Chart deployment tutorial](/docs/continuous-delivery/deploy-srv-diff-platforms/helm/helm-cd-quickstart) for details on adding Helm Charts to a Stage's **Service Definition**.
+   Go to [Helm Chart deployment tutorial](/docs/continuous-delivery/deploy-srv-diff-platforms/helm/helm-cd-quickstart) for details on adding Helm Charts to a Stage's **Service Definition**.
```
Comment (roshnisarangadharan) [inline docs/platform/11_Triggers/trigger-pipelines-on-new-helm-chart.md:83]:
```suggestion
   Go to [Helm Chart deployment tutorial](/docs/continuous-delivery/deploy-srv-diff-platforms/helm/helm-cd-quickstart) for details on adding Helm Charts to a stage's **Service Definition**.
```

File: docs/platform/11_Triggers/trigger-pipelines-on-new-helm-chart.md [hunk 3]
```diff
@@ -58,7 +62,7 @@ When you add the Helm Chart to Harness as a Manifest, you have different options
 
 ![](./static/trigger-pipelines-on-new-helm-chart-04.png)
 * **Fixed Value:** if you use [Fixed Value](../20_References/runtime-inputs.md) for **Chart Version** (for example, `0.1.4`), Helm Chart Triggers will work, but Harness will not select the latest chart version. Instead, Harness will select the hardcoded chart version in **Chart Version** (`0.1.4`).
-* **Runtime Input:** if you use [Runtime Input](../20_References/runtime-inputs.md) for **Chart Version**, you can enter the version to use in your Trigger as part of the Trigger Pipeline Inputs. See [Select Pipeline Inputs](trigger-pipelines-on-new-helm-chart.md#step-4-select-pipeline-inputs) below.
+* **Runtime Input:** if you use [Runtime Input](../20_References/runtime-inputs.md) for **Chart Version**, you can enter the version to use in your Trigger as part of the Trigger Pipeline Inputs. Go to [Select Pipeline Inputs](trigger-pipelines-on-new-helm-chart.md#step-4-select-pipeline-inputs) below.
```
Comment (roshnisarangadharan) [inline docs/platform/11_Triggers/trigger-pipelines-on-new-helm-chart.md:65]:
```suggestion
* **Runtime Input:** if you use [Runtime input](../20_References/runtime-inputs.md) for **Chart Version**, you can enter the version to use in your trigger as part of the **Trigger Pipeline Inputs**. Go to [Select pipeline inputs](trigger-pipelines-on-new-helm-chart.md#step-4-select-pipeline-inputs) below.
```

**Review by roshnisarangadharan — APPROVED** [review]
Just a few case related edits. You might find such instances in other parts of the same doc. So, ignore if you don't want to get in to such edits now.

**Pull request merged**